cmd/limactl/copy.go

```diff
@@ -67,6 +82,9 @@ func copyAction(cmd *cobra.Command, args []string) error {
 
 	if verbose {
 		scpFlags = append(scpFlags, "-v")
+		if useRsync {
+			scpFlags = append(scpFlags, "--progress")
```

Comment:
We cannot use variable `scpFlags` to keep rsync flags. This will confuse the next developer. Renaming the variable to toolFlags will avoid this.